Header header

float64 error_roll
float64 error_pitch
float64 error_yaw           # Errore yaw relativo in radianti

float64 error_distance
float64 error_depth

float64 error_surge_speed   # Errore surge assoluto in m/s
float64 error_sway_speed    # Errore sway assoluto in m/s